pandas/_libs/tslibs/strptime.pyx

```diff
@@ -415,13 +415,17 @@ def array_strptime(
                     creso = state.creso
                 value = npy_datetimestruct_to_datetime(creso, &dts)
                 if out_local == 1:
-                    # Store the out_tzoffset in seconds
-                    # since we store the total_seconds of
-                    # dateutil.tz.tzoffset objects
+                    nsecs = out_tzoffset * 60
```

Comment:
Curious why do we need to store the offset in seconds? It seems elsewhere we store it in minutes

Reply:
in array_to_datetime we get these from `nsecs = tz.utcoffset(None).total_seconds()`.  this chunk of code is patterned off of that (and eventually intended to be de-duplicated with that)